pkg/accounting/tally/tally.go

```diff
@@ -63,11 +63,11 @@ func (t *Service) Run(ctx context.Context) (err error) {
 	}
 }
 
-// Tally calculates data-at-rest once
+// Tally calculates data-at-rest and bandwidth usage once
```

Comment:
You can remove the bandwidth part since we're doing that outside of the tally service